app/src/main/kotlin/com/intive/picover/parties/view/PartyDetailsScreen.kt

```diff
@@ -0,0 +1,54 @@
+package com.intive.picover.parties.view
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalLifecycleOwner
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.intive.picover.common.loader.PicoverLoader
+import com.intive.picover.main.theme.Typography
+import com.intive.picover.parties.model.Party
+import com.intive.picover.parties.state.PartyDetailsState
+import com.intive.picover.parties.viewmodel.PartyDetailsViewModel
+
+@Composable
+fun PartyDetailsScreen(
+	viewModel: PartyDetailsViewModel,
+) {
+	LocalLifecycleOwner.current.lifecycle.addObserver(viewModel)
+	when (val state = viewModel.partyDetails.value) {
+		is PartyDetailsState.Loading -> {
+			PicoverLoader()
+		}
+
+		is PartyDetailsState.Loaded -> {
+			LoadedContent(party = state.party)
+		}
+	}
```

Comment:
conditions could be in one line, I mean:
```
when (val state = viewModel.partyDetails.value) {
		is PartyDetailsState.Loading -> PicoverLoader()
		is PartyDetailsState.Loaded -> LoadedContent(party = state.party)
	}
```

Reply:
Done